File: Back/PilatesAPI/Entities/Medida.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PilatesAPI.Entities
{


    public class Medida : Entity
    {
        public Medida(int altura, int peso, int cintura, int busto, int biceps, int coxa, int panturrilha, int alunoId)
        {
            Altura = altura;
            Peso = peso;
            Cintura = cintura;
            Busto = busto;
            Biceps = biceps;
            Coxa = coxa;
            Panturrilha = panturrilha;
            AlunoId = alunoId;

            CriadoPor = "Andrey Teste";
            EditadoPor = "Andrey Teste";
            EditadoEm = DateTime.Now;
        }

        public int Id { get; set; }
        public int Altura { get; set; }
        public int Peso { get; set; }
        public int Cintura { get; set; }
        public int Busto { get; set; }
        public int Biceps { get; set; }
        public int Coxa { get; set; }
        public int Panturrilha { get; set; }
        public int AlunoId { get; set; }
        public Aluno Aluno { get; set; }

    }
}

File: Front/EspacoLuz/src/app/models/Medida.ts
import { Aluno } from './Aluno';

export interface Medida {
  id: number;
  data: string;
  altura: string;
  peso: string;
  cintura: string;
  busto: string;
  biceps: string;
  coxa: string;
  panturrilha: string;
}

export interface medidaPostRequest {
  alunoId: number;
  altura: number;
  peso: number;
  cintura: number;
  busto: number;
  biceps: number;
  coxa: number;
  panturrilha: number;
}

export interface medidaPutRequest {
  altura: number;
  peso: number;
  cintura: number;
  busto: number;
  biceps: number;
  coxa: number;
  panturrilha: number;
}

export interface responseMedida {
  value: Medida[];
  statusCode: number;
  contentType?: any;
}

export interface ResponseMedidas {
  id: number;
  altura: number;
  peso: number;
  cintura: number;
  busto: number;
  biceps: number;
  coxa: number;
  panturrilha: number;
  alunoId: number;
  aluno?: any;
  criadoPor: string;
  criadoEm: Date;
  editadoPor: string;
  editadoEm: Date;
  notifications: any[];
  isValid: boolean;
}
